qiskit/providers/aer/noise/noise_model.py

```diff
@@ -698,19 +695,24 @@ def from_dict(noise_dict):
                 raise NoiseError("Invalid error type: {}".format(error_type))
         return noise_model
 
-    def _instruction_names(self, instructions):
-        """Return a list of instruction name strings for input instructions."""
+    def _instruction_names_labels(self, instructions):
+        """Return two lists of instruction name strings and label strings."""
```

Comment:
Returns one list of pairs. By the way this could be directly implemented accordingly, instead of creating two lists and zipping them, but two lists + zip is also OK.